extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -71,24 +79,69 @@ const systemShareSupported = (viewer, platform) => {
   return 'share' in navigator && !isChromeWebview;
 };
 
+/**
+ * @private
+ * @param {!Element} element
+ * @param {Array<MutationRecord>} mutations
+ * @param {string} prevTypeValue
+ * @return {!JsonObject|undefined}
+ */
+const updateTypeConfig = (element, mutations, prevTypeValue) => {
+  let typeUpdated;
+  let mutatedEligibleAttribute;
+
+  // Check all mutations since we want to catch any 'data-param-*' attributes
+  mutations.forEach((mutation) => {
+    if (
+      ['type', 'data-target', 'data-share-endpoint'].includes(
```

Comment:
This is actually much slower than `attrName === 'type' || attrName === 'data-target' …`

Reply:
Thanks, updated!